python/ray/_private/node.py

```diff
@@ -270,7 +270,16 @@ def __init__(
 
         # Pick a GCS server port.
         if head:
-            gcs_server_port = os.getenv(ray_constants.GCS_PORT_ENVIRONMENT_VARIABLE)
+            gcs_server_port_environment = os.getenv(
```

Comment:
ray_constants.py has a helper function called `env_integer()`, lets use that.

Reply:
Thankyou, I will change it.

Reply:
@jjyao I have completed it